dashboard_backend/src/features/dashboard/dashboard_service.py

```diff
@@ -269,11 +269,23 @@ def update_settings(self, data):
                 setattr(self.chronos, key, value)
         return reponse
 
+    def turn_off_devices(self):
```

Comment:
hmm... we never want to turn off all devices. I'm not sure why you are trying that. At the moment when the season is switched the backend is supposed to:
change one season valve state (these are controlled by relays that are toggled on and off)
wait about a minute and then change the other season valve state
The above should never be interrupted... that's what the lockout is for.
My request at the moment is that if no valve/relay can be toggled (the edge server rejects the request for some reason) then the interface indicates that a season switch can't occur.

Reply:
As mentioned on slack, the mode switch actually does switch off all devices. Apologies.

![image (8)](https://github.com/user-attachments/assets/3a69d0a6-42f0-4a00-9adf-3e6920430fc8)